src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -72,9 +69,13 @@
 }
 
 .action {
-  align-items: center;
+  align-items: stretch;
   display: flex;
   flex: 0 0 auto;
   justify-self: flex-end;
-  margin: var(--calcite-spacing-quarter) var(--calcite-spacing-half);
+  margin: 0;
+}
+
+.action slot[name="secondary-action"]::slotted(calcite-action) {
```

Comment:
This one is a bit complex if we could simplify the selector. If not, no worries.

Reply:
I'll see if it works without the `.action`.

Reply:
Turns out, I didn't even need that block.